dll/win32/shell32/shellrecyclebin/recyclebin_v5_enumerator.c

```diff
@@ -34,6 +37,10 @@ RecycleBin5File_RecycleBinFile_QueryInterface(
         *ppvObject = &s->recycleBinFileImpl;
     else if (IsEqualIID(riid, &IID_IRecycleBinFile))
         *ppvObject = &s->recycleBinFileImpl;
+    else if (IsEqualIID(riid, &IID_IExtractIconA) || IsEqualIID(riid, &IID_IExtractIconW))
+    {
+        return SHCreateFileExtractIconW(s->FullName, 0, riid, ppvObject);
```

Comment:
Does the recycler store any file attributes by the way?
If not, please use `FILE_ATTRIBUTE_NORMAL` instead of zero (for the 2nd parameter).

Reply:
Done